src/ecr/process.cr

```diff
@@ -1,2 +1,14 @@
 require "ecr/processor"
-puts ECR.process_file(ARGV[0], ARGV[1])
+
+filename = ARGV[0]
+buffer_name = ARGV[1]
+
+begin
+  puts ECR.process_file(filename, buffer_name)
+rescue ex : Errno
+  raise ex unless {Errno::ENOENT, Errno::EISDIR}.includes?(ex.errno)
```

Comment:
That makes sense when the method body is long, but here's it's just two statements, making the `if` clearer. Once it gets to 5-7+ lines i'd agree with you.